htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -1791,6 +1791,53 @@ public function validate($user, $force_number = '', $idwarehouse = 0, $notrigger
 			return -1;
 		}
 
+		// Check for mandatory fields in thirdparty (defined into setup)
+		if (!empty($this->thirdparty) && is_object($this->thirdparty)) {
```

Comment:
In the setup, the option is "fields mandatory" to validate a sale invoice.
Here we are on a vendor invoice.